source/common/stream_info/stream_info_impl.h

```diff
@@ -443,6 +451,9 @@ struct StreamInfoImpl : public StreamInfo {
   FilterStateSharedPtr legacy_upstream_filter_state_;
   std::string route_name_;
   absl::optional<uint32_t> attempt_count_;
+  // TODO: Check if the owner of this storage outlives the StreamInfo. We should only copy the
+  //  string if it could outlive the StreamInfo.
```

Comment:
nit -- extra space before 'string' at beginning of line.

Rather than an unowned `TODO: ` -- use `TODO(agrawroh)` or generate an issue for it and use `TODO(#ISSUE_NUMBER)`

Reply:
Changed it to `TODO(agrawroh)`.